cmd/vic-machine/common/utils.go

```diff
@@ -31,3 +34,15 @@ func LogErrorIfAny(clic *cli.Context, err error) error {
 	log.Errorf("%s %s failed: %s\n", clic.App.Name, clic.Command.Name, errors.ErrorStack(err))
 	return cli.NewExitError("", 1)
 }
+
+// CheckUnsupportedChars returns an error if string contains special characters
+// https://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=2046088
+func CheckUnsupportedChars(s string) error {
+	st := []byte(s)
+	re := regexp.MustCompile(`%|&|\*|\$|#|@|!|\\|/|:|\?|"|<|>|;|'|\|`)
+
+	if v := re.FindIndex(st); v == nil {
+		return nil
+	}
+	return fmt.Errorf("invalid characters in %q", s)
```

Comment:
Will that be better to use unsupported character, instead of invalid?
In addition, it will be better to list all unsupported character here. So user knows what's wrong easily.